deps/wazuh_testing/wazuh_testing/tools/macos_log/macos_utils.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+def compose_settings(__type="", __level="", __predicate="", __is_sierra=False):
```

Comment:
The name `compose_setting` maybe is too generic, especially if it is moved to logcollector. It should be renamed to `compose_log_process` or something more specific.
```suggestion
def compose_log_process(type=None, level=None, predicate=None, is_sierra=False):
```